### ISSUE TITLE
Release a build of the package

### ISSUE THREAD

**Issue by davidcurrie2001** [ices-tools-dev/RDBEScore#218]
A participant at this week's WKRDBESRaiseStock workshop asked if we could release an up-to-date build of the package so that people who can't install packages from GitHub would be able to use it.

**Comment by davidcurrie2001** [comment]
I don't think we can wait until 2027 to release the package.  ICES also uses the R-universe repository https://ices-tools-prod.r-universe.dev/packages which has fewer requirements than CRAN so that might be an easier target to aim for.

**Comment by rix133** [comment]
For those who don't know the RDBEScore is allready some years at  R-universe repository https://ices-tools-prod.r-universe.dev/RDBEScore the main branch should be automatically published there so so eg #211 should be the latest fixed issue there

**Comment by davidcurrie2001** [comment]
Great - we can point people to there as an alternative to installing from GitHub then :-)

**Comment by rix133** [comment]
@colinpmillar  would it be possible to make it unfiden form search results: https://ices-tools-prod.r-universe.dev/RDBEScore

**Comment by colinpmillar** [comment]
Hi @rix133 yes! just setting that up now, hopefully later today it should be visible.
Thanks for the heads up!

**Comment by colinpmillar** [comment]
I have created an ices-tools-dev r-universe.  But perhaps, a better way is to move the package to ices-tools-prod ?? if it is getting more mature?
